x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/enrich/AbstractLookupService.java

```diff
@@ -127,14 +126,25 @@
  * </p>
  */
 public abstract class AbstractLookupService<R extends AbstractLookupService.Request, T extends AbstractLookupService.TransportRequest> {
+
+    /**
+     * Field for Positions in lookup operations. Contains an IntBlock of positions.
+     */
+    public static final EsField LOOKUP_POSITIONS_FIELD = new EsField(
```

Comment:
EsField normally represents physical fields inside ES indices. Is there a different way we could express this information?

The most standard attribute type is a `ReferenceAttribute`. Can we use that here, or do we need a more specialized kind of attribute? (Which is okay to add if needed.)